static/src/javascripts/bootstraps/standard/main.js

```diff
@@ -110,6 +110,27 @@ define([
             }
         });
 
+        /*
+         *  Interactive bootstraps.
+         *
+         *  Interactives are content, we want them booting as soon (and as stable) as possible.
+         */
+
+        if (
+            config.switches.bootInteractivesFromMain &&
+            /Article|Interactive|LiveBlog/.test(config.page.contentType)
+        ) {
+            $('figure.interactive').each(function (el) {
```

Comment:
Once its proved to work in `main.js` it gets deleted from the other module. They will not co-exist for long. Just keeping it there for now so we can switch between implementations.